packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,249 @@ def _clean_html(self, html: str) -> str:
                     )
-                    raise
+                    break
+
+                # Move to next page
+                start += limit
+                logger.debug(f"Moving to next page with start={start}")
+
+            except Exception as e:
+                logger.error(
+                    f"Failed to fetch content from space {self.config.space_key}: {e!s}"
+                )
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)